homeassistant/components/local_file/strings.json

```diff
@@ -1,4 +1,42 @@
 {
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
+    },
+    "error": {
+      "not_readable_path": "The provided path to the file can not be read"
+    },
+    "step": {
+      "user": {
+        "data": {
+          "name": "[%key:common::config_flow::data::name%]",
+          "file_path": "File path"
+        },
+        "data_description": {
+          "name": "Name for the created entity.",
+          "file_path": "Path to file, see documentation for further details."
```

Comment:
What would "further details" entail? When I open this I would expect to just fill in an absolute path. Should we maybe use the description to at least explain why someone would need the docs? It now sounds a bit like "ah, here, you will need it" (hard to explain)

Reply:
Any suggestion? The point is that people needs to be aware it needs to be added to the allowed paths. Should we link to the relevant section?

Reply:
Ah yes, now I know this, yes, it's indeed something anyone must know before being able to setup.
Can we detect if something is added to allowed paths? As in, if someone adds a non existent path to allowed paths, does that raise somewhere else or isthat allowed?

Reply:
Copied what's there for the service which I guess makes sense.

Reply:
I would say something like "Be sure to add the path of the file to the allowed paths, you can read more about this in the documentation"

Reply:
Fixed